cmd/main.go

```diff
@@ -39,24 +46,26 @@ var (
 )
 
 func init() {
-	_ = clientgoscheme.AddToScheme(scheme)
+	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 
-	// +kubebuilder:scaffold:scheme
+	//+kubebuilder:scaffold:scheme
 }
 
 func main() {
-	var metricsAddr string
-	var enableLeaderElection bool
 	var fastlyAPIToken string
 	var fastlyPlatformTLSConfiguration string
 	var clusterName string
 	var enablePausedStatusCron bool
 	var pausedStatusCron string
 	var maxRetryCount int
-
-	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
-	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
-		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
+	var metricsAddr string
+	var enableLeaderElection bool
+	var probeAddr string
+	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
```

Comment:
This would be a breaking change we should avoid.


```suggestion
	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
```

Reply:
Let's keep that in mind then to add to the next release notes, but no need to change as it makes sense to follow convention.